src/theme/src/theme-tools.js

```diff
@@ -0,0 +1,47 @@
+export function get(obj, path, fallback) {
+  const keys = path && path.split ? path.split('.') : [path]
+
+  let value = obj
+  for (const key of keys) {
+    value = value ? value[key] : undefined
+  }
+
+  return value === undefined ? fallback : value
```

Comment:
Recursing here _one_ more time would give us token aliases, which I think would be a pretty nice feature.

Reply:
True! Though you could also define the alias with a hard value instead of a token string (you should have access to it when defining the theme anyway)